src/app/devices/sensor-details/sensor-detail.component.ts

```diff
@@ -54,19 +54,33 @@ export class SensorDetailComponent {
         });
     }
 
-    cancel() {
+    public cancel(): void {
         this._navigateToList();
     }
 
-    remove() {
+    public remove(): void {
         this.sensorDetailService
             .delete(this.sensor)
             .subscribe(() => {
                 this._navigateToList();
             });
     }
 
-    _navigateToList() {
+    /**
+     * Make sure that servo and executor checkboxes are both not checked at one time.
+     * @param {string} chkType - Checkbox, what was changed. Possible variants: 'executor', 'servo'
+     */
+    public validate(chkType: string): void {
+        if (this.sensor.executor && this.sensor.servo) {
```

Comment:
I would rather split this into two functions **validateServo** and **validateExecutor**, as it is possible to use them separately in template